docs/cn/compute/Spark-On-Kubernetes.md

```diff
@@ -0,0 +1,202 @@
+
+##基本设置
+
+首先，我们准备一个Spark Docker镜像，其中包括Alluxio客户端和任何其他必需的jar文件。
+在所有Kubernetes节点上都需提供此镜像。
+
+###下载二进制文件
+
+[下载](https://spark.apache.org/downloads.html)所需的Spark版本。
+对于`spark-submit`命令和使用Alluxio所含的Dockerfile编译Docker镜像
+我们都使用预生成的二进制文件，。
+>注:下载为Hadoop预制的软件包
+
+```console
+$ tar -xf spark-2.4.4-bin-hadoop2.7.tgz
+$ cd spark-2.4.4-bin-hadoop2.7
+```
+###编译Spark Docker镜像
+
+从Alluxio Docker镜像中提取Alluxio客户端jar:
+
+```console
+$ id=$(docker create alluxio/alluxio:{{site.ALLUXIO_VERSION_STRING}})
+$ docker cp $id:/opt/alluxio/client/alluxio-{{site.ALLUXIO_VERSION_STRING}}-client.jar \
+  - > alluxio-{{site.ALLUXIO_VERSION_STRING}}-client.jar
+$ docker rm -v $id 1>/dev/null
+```
+
+添加所需的Alluxio客户端jar并构建用于Spark驱动程序和执行程序pods的Docker镜像。
+从Spark发行版目录运行以下命令以添加Alluxio客户端jar。
+
+```console
+$ cp <path_to_alluxio_client>/alluxio-{{site.ALLUXIO_VERSION_STRING}}-client.jar jars/
+```
+>注意:任何复制到`jars`目录的jar文件在编译时都会被包含到Spark Docker镜像中。
+
+编译Spark Docker镜像
+
+```console
+$ docker build -t spark-alluxio -f kubernetes/dockerfiles/spark/Dockerfile .
+```
+>注意:确保所有节点(spark-driver和spark-executor pods将运行的所在节点) 
+都有该镜像。
+
+##示例
+
+本节说明如何使用编译的Docker镜像来发起一个以Alluxio作为数据源的Spark作业。
+
+###短路操作
+
+短路访问使Spark执行器中的Alluxio客户端可以直接访问
+主机上的Alluxio worker存储。
+因为不通过网络堆栈来与Alluxio worker通信，这样可以提高性能。
+
+如果在部署Alluxio时未按照指令设置域套接字
+][本页]({{ '/en/deploy/Running-Alluxio-On-Kubernetes.html' | relativize_url}}＃short-circuit-access)，则
+可以跳过将`hostPath`卷挂载到Spark执行器步骤。
+
+如果在运行Alluxio worker进程的主机上将域套接字位置设置为
+`/tmp/alluxio-domain`，并且Alluxio配置为
+`alluxio.worker.data.server.domain.socket.address=/opt/domain`，使用以下Spark
+配置将`/tmp/alluxio-domain`挂载到Spark执行器pod中的`/opt/domain`。
+下一节中的`spark-submit`命令包含这些属性。
+
+取决于你的设置，Alluxio worker上的域套接字可以是`hostPath`卷或`PersistententVolumeClaim`两种之一。可以在此处找到有关如何配置Alluxio worker以使用短路操作的更多详细信息 
+({{ '/en/deploy/Running-Alluxio-On-Kubernetes.html#short-circuit-access' | relativize_url}})。
+这两个选项的spark-submit参数将有所不同。
+可以在以下Spark文档中找到有关如何将卷挂载到Spark执行器的更多[信息](https://spark.apache.org/docs/2.4.4/running-on-kubernetes.html#using-kubernetes-volumes)。
+
+{% navtabs domainSocket %}
+  {% navtab hostPath %}
+  如果使用的是hostPath域套接字，则应将以下属性传递给Spark:
+  
+  ```properties
+  spark.kubernetes.executor.volumes.hostPath.alluxio-domain.mount.path=/opt/domain
+  spark.kubernetes.executor.volumes.hostPath.alluxio-domain.mount.readOnly=true
+  spark.kubernetes.executor.volumes.hostPath.alluxio-domain.options.path=/tmp/alluxio-domain
+  spark.kubernetes.executor.volumes.hostPath.alluxio-domain.options.type=Directory
+  ```
+ 
+{% endnavtab %}
+  {% navtab PersistententVolumeClaim %}
+  如果使用的是PersistententVolumeClaim域套接字，则应将以下属性传递给Spark:
+  
+  ```properties
+  spark.kubernetes.executor.volumes.persistentVolumeClaim.alluxio-domain.mount.path=/opt/domain \
+  spark.kubernetes.executor.volumes.persistentVolumeClaim.alluxio-domain.mount.readOnly=true \
+  spark.kubernetes.executor.volumes.persistentVolumeClaim.alluxio-domain.options.claimName=<domainSocketPVC name>
+  ```
+  
+  {% endnavtabs %}
```

Comment:
`{% endnavtab %}`

Reply:
these two lines are still incorrect. they should be:
```
  {% endnavtab %}
{% endnavtabs %}
```
the first line does not have a `s` to match with `{% navtab PersistententVolumeClaim%}` in line 107
the second line should not be indented to match with with `{% navtabs domainSocket %}` in line 95